plugins/azure/virtualmachines/vmDiskDoubleEncryption.js

```diff
@@ -0,0 +1,57 @@
+var async = require('async');
+
+var helpers = require('../../../helpers/azure/');
+
+module.exports = {
+    title: 'VM Disk Double Encryption',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that VM disks are encrypted at rest using both platform and customer managed keys.',
+    more_info: 'Using double encryption for VM disks adds an extra layer of protection using a different encryption algorithm/mode at the infrastructure layer using platform managed encryption keys and provides an additional level of security if one of the keys is compromised.',
+    recommended_action: 'Enable double encryption for all VM disks.',
```

Comment:
i think we cannot enabled double encryption after vm disk is created 
<img width="900" alt="Screenshot 2023-12-29 at 1 45 15 PM" src="https://github.com/aquasecurity/cloudsploit/assets/113519745/1d141c90-2ee3-43ed-8038-1dcbb46cbf18">